geowave/src/test/scala/geotrellis/spark/io/geowave/GeoWaveFeatureRDDReaderSpec.scala

```diff
@@ -93,8 +62,8 @@ class GeoWaveFeatureRDDReaderSpec extends FunSpec { self: Suite =>
         .map { x: Int => Feature(Point(x, 40), Map[String, Any]()) }
         .toArray
       val featureRDD = sc.parallelize(features)
-      val zookeeper = "localhost:20000"
-      val instanceName = "AccumuloInstance"
+      val zookeeper = "localhost:21810"
```

Comment:
Can we use Accumulo Mock instance? (iterators would be availabl, if they would be included as a dep in tests)